torch/csrc/jit/mobile/import.cpp

```diff
@@ -87,7 +84,28 @@ void print_unsupported_ops_and_throw(
 void parseMethods(
     const std::vector<IValue>& vals,
     mobile::CompilationUnit& mcu) {
-  for (const auto& element : vals) {
+  TORCH_CHECK(
```

Comment:
If we're supporting old models (per lines 90-98, below) then I don't think we want this check, right?

Reply:
Updated the message to just check the size to avoid access error later.